libr/core/cio.c

```diff
@@ -432,29 +419,25 @@ static RCoreFile * r_core_file_set_first_valid(RCore *core) {
 
 R_API int r_core_block_read(RCore *core) {
 	if (!core->file && !r_core_file_set_first_valid (core)) {
-		memset (core->block, core->io->Oxff, core->blocksize);
+		memset (core->block, 0xff, core->blocksize);		//io->Oxff
 		return -1;
 	}
 	if (core->file && core->switch_file_view) {
-		r_io_use_desc (core->io, core->file->desc);
-		r_core_bin_set_by_fd (core, core->file->desc->fd); //needed?
+		r_io_desc_use (core->io, core->file->desc->fd);
+		r_core_bin_set_by_fd (core, core->file->desc->fd);	//needed?
 		core->switch_file_view = 0;
-	} else	{
-		r_io_use_fd (core->io, core->io->raised); //possibly not needed
 	}
 	return r_io_read_at (core->io, core->offset, core->block, core->blocksize);
 }
 
 R_API int r_core_read_at(RCore *core, ut64 addr, ut8 *buf, int size) {
 	if (!core || !core->io || !core->file || !core->file->desc || size < 1) {
-		if (core && core->io && size > 0) {
-			memset (buf, core->io->Oxff, size);
+		if (core && core->io && size > 0) {			//wtf
```

Comment:
Looks OK. May be splitting `|| size < 1` from the `if` will help to understand for a new people.